frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -156,7 +151,8 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
         endBuffer: true,
         startScrub: true,
         endScrub: true,
-        setErrorPlayerState: (show: boolean) => ({ show }),
+        setPlayerError: (reason: string) => ({ reason }),
+        clearPlayerError: true,
```

Comment:
"show" didn't really mean show, so now we store the reason and have a `clear` action